pkg/apiserver/query/types.go

```diff
@@ -52,29 +33,30 @@ type Pagination struct {
 	// items per page
 	Limit int
 
-	// page number
-	Page int
+	// offset
+	Offset int
 }
```

Comment:
Why are we using **Offset** instead of **Page**?

Reply:
Page usually start at 1, offset start at 0.